### DISCUSSION TITLE
blog/00-academic-astro

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#64] [Announcements]
# blog/00-academic-astro

Features, enhancements, and changes.

https://astro-micro-academic.vercel.app/blog/00-academic-astro

**Comment by elgyro** [comment]
Hell yeah!!

**Reply by elgyro** [reply]
works👍

**Reply by elgyro** [reply]
@elgyro sup?

**Reply by elgyro** [reply]
interesting @luciancah     
$a = b + 1, a = \frac{1}{x_1}$

$$a = \sum_{n=1}^{22} \sqrt{a + b_n}$$

**Reply by trevortylerlee** [reply]
Heck yes!
